Configuration/TCA/tx_variables_marker.php

```diff
@@ -100,7 +100,7 @@
             'l10n_mode' => 'exclude',
             'label' => 'LLL:EXT:lang/locallang_general.xlf:LGL.starttime',
             'config' => [
-                'type' => 'input',
+                'type' => 'inputDateTime',
```

Comment:
sicher, dass es so funktioniert? In der Doku ist es etwas anders (mit `renderType`).
https://docs.typo3.org/m/typo3/reference-tca/master/en-us/ColumnsConfig/Type/Input.html#properties-rendertype-inputdatetime

Reply:
Sorry, war zu schnell